frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_permissions.py

```diff
@@ -11,6 +11,14 @@
 from . import GET, PUT, editable_copr, get_copr
 
 
+@apiv3_ns.route("/project/permissions/can_build_in/<ownername>/<projectname>")
+@api_login_required
+def can_build_in(ownername, projectname):
```

Comment:
missing doc string

Reply:
Something weird happened and you commented on an outdated code. The docstring is there.